app/services/requirements/backdate_checker.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Requirements
+  class BackdateChecker
+    def initialize(backdate)
+      @backdate = backdate
+    end
+
+    def pre_submit_issues
+      issues = []
+
+      if in_the_future?
+        issues << Issue.new(:backdate, :in_the_future)
```

Comment:
Minor, but I wonder if we should also prevent publishers from backdating to 'today'? Probably doesn't matter very much, but it might be worth checking with @benhazell to see if it's sensible.

Reply:
I had pondered this, and wondered if possibly they would want to backdate it to today and then actually publish it at a future date?  Not sure if it's a thing.